### ISSUE TITLE
'migro' is not recognized as an internal or external command

### ISSUE THREAD

**Issue by sadashiv-sumasoft** [uploadcare/migro#107]
I have successfully installed uploadcare-migro on my Windows machine, and I also have Python version 3.10, but when I hit the
migro <mypublickey> < textfile path > It is throwing an error called Migro that is not recognized as an internal or external command.

I have performed the following steps to resolve the issue, but they did not work.
1. I visited the path where Migro has been installed, but I did not find any executable files to add the path to the system environment variable.
2. Also, I did not find the migro.exe file.

Please let me know if I am doing anything wrong.
Basically, I wanted to copy the files from one uploadcare project to another. If this way is not available, tell me any smart way I can achieve this.
